.github/workflows/main.yml

```
@@ -15,20 +16,66 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)